kuma/wiki/search.py

```diff
@@ -308,8 +308,8 @@ def reindex_all(cls, chunk_size=500, index=None, percent=100):
             chord_flow(pre_task, index_tasks, post_task).apply_async()
 
         message = _(
-            'Indexing {total} documents into {n} chunks of size {size} into '
-            'index {index}.'.format(total=total, n=total_chunks,
-                                    size=chunk_size,
-                                    index=index.prefixed_name))
+            'Indexing %(total)s documents into %(total_chunks)s chunks of '
+            'size %(size)s into index %(index)s.' % {
+                'total': total, 'total_chunks': total_chunks,
+                'size': chunk_size, 'index': index.prefixed_name})
```

Comment:
@robhudson Sure, here's the fixed version:

```python
message = _(
    'Indexing %(total)d documents into %(total_chunks)d chunks of '
    'size %(size)d into index %(index)s.' % {
        'total': total,
        'total_chunks': total_chunks,
        'size': chunk_size,
        'index': index.prefixed_name,
    }
)
```

Note the use of `%d` as the decimal integer representation.